src/ProtocolStreamWriter.php

```diff
@@ -31,16 +31,7 @@ public function write(Message $msg)
         $totalBytesWritten = 0;
 
         while ($totalBytesWritten < $msgSize) {
-            error_clear_last();
-            $bytesWritten = @fwrite($this->output, substr($data, $totalBytesWritten));
-            if ($bytesWritten === false) {
-                $error = error_get_last();
-                if ($error !== null) {
-                    throw new RuntimeException('Could not write message: ' . error_get_last()['message']);
-                } else {
-                    throw new RuntimeException('Could not write message');
-                }
-            }
+            $bytesWritten = fwrite($this->output, substr($data, $totalBytesWritten));
```

Comment:
100% agree with this change. I'm not even sure if the error handling worked before. It was written when we didn't have an ErrorHandler to convert notices to exceptions.